pyproject.toml

```diff
@@ -66,8 +65,8 @@ aws = [
     "boto3==1.34.106",
     "dbt-redshift>=1.8.1, <1.10",
     "minio>=7.0, <8.0",
-    # We need to pin this to 2.x because some code depends on 2.x-only functionality.
-    "awswrangler>=2.20.1, <3.0",
+    # Updated awswrangler constraint
```

Comment:
This comment doesn't make sense - it raises the questions: "updated from what?" and "so what, why would I care this random package was at one time updated?" etc. and so it's only confusing the reader, can you remove it? You probably meant to add this as a comment to a commit or in PR description.